.github/workflows/build.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         uses: actions/cache@v2
         with:
           path: '**/node_modules'
-          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}-v1
       
```

Comment:
Added v1 to the key (and v2 to the build-v2 workflow) to differentiate the node module cache between the two

Reply:
I see you have build_v2.yml, would it hurt anything if you renamed build.yml -> build_v1.yml 
This way people unfamiliar with how this repo is setup can explicitly know which one is for v1 and v2

Reply:
Additionally anything in this file that just says "build" may also benefit from having the `_v1` added to it

Reply:
good idea, made the changes